cibuild.sh

```diff
@@ -309,7 +309,7 @@ fi
 # warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.
 # warning MSB3026: Could not copy "XXX" to "XXX". Beginning retry 1 in 1000ms.
 # warning AL1053: The version '1.2.3.4-foo' specified for the 'product version' is not in the normal 'major.minor.build.revision' format
-_NOWARN="MSB3277;MSB3026;AL1053"
+_NOWARN="MSB3276;MSB3277;MSB3026;AL1053"
```

Comment:
why is it compiling against a 2.0 mscorlib?
if that's expected, you should also add a comment above why this warning was disabled